src/utilities/error.c

```diff
@@ -89,6 +89,22 @@ hypre_error_handler(const char *filename, HYPRE_Int line, HYPRE_Int ierr, const
    hypre__global_error = err;
 }
 
+/*--------------------------------------------------------------------------
+ * hypre_error_handler_destroy
+ *--------------------------------------------------------------------------*/
+
+void
+hypre_error_handler_destroy(void)
```

Comment:
I would call this `hypre_error_clear_messages` instead.  That's all it really does.  There is no corresponding `create` call that "creates" an error handler.

Reply:
Done in [ff71293](https://github.com/hypre-space/hypre/pull/1190/commits/ff7129391c8cb35566c41a29e0203237bfca4a0b)